src/Commands/HearthCommand.php

```diff
@@ -93,188 +93,90 @@ public function handle()
             (new Filesystem())->ensureDirectoryExists(app_path('Policies'));
             (new Filesystem())->ensureDirectoryExists(app_path('Rules'));
             (new Filesystem())->ensureDirectoryExists(app_path('View/Components'));
+            (new Filesystem())->ensureDirectoryExists(lang_path('fr'));
```

Comment:
Do you need to create a new instance of Filesystem for every call? I wonder if it would be possible to have a protected or private property assigned a Filesystem instance.